internal/sandbox/sandbox.go

```diff
@@ -94,6 +95,12 @@ type Sandbox interface {
 	//
 	// Once called the Sandbox cannot be used again.
 	Clean() error
+
+	// CopyToHost copies a path in the sandbox (src) to a path in the host (dest).
+	// It will fail if the src path does not exist in the sandbox.
+	// See https://docs.podman.io/en/latest/markdown/podman-cp.1.html for
+	// semantics of src and dest paths
+	CopyToHost(src, dest string) error
```

Comment:
I suggest adding a comment that data coming out of a sandbox is unsafe and must not be trusted without correct validation.